packages/enzyme/src/Debug.js

```diff
@@ -43,15 +44,19 @@ function propString(prop) {
     return `{${inspect(prop)}}`;
   }
   if (typeof prop === 'object') {
+    if (options.verbose) {
+      return `{${CircularJSON.stringify(prop)}}`;
```

Comment:
Why not `inspect(prop)`?

Reply:
let me test this. I assumed it wouldn't do what I wanted it to.

Reply:
This is currently the difference between the two libs:

```jsx
{ 
  inspect:      <div data-json={{ a: [ 1, 3, { true: true } ], b: false, c: { d: 'f' }, d: [ 1, 3, { true: true } ] }} data-arry={[ 1, 2, { f: { d: 'f' } } ]}>
  circularJSON: <div data-json={{"a":[1,3,{"true":true}],"b":false,"c":{"d":"f"},"d":"~a"}} data-arry={[1,2,{"f":{"d":"f"}}]}>
}
```

Seems like the inspect one looks more like real code, rather than json. But It will definitely crash if used against circular structures since it just prints them out directly. If I pass in something like:

```js
a = {};
a.b = {};
a.c = a.b;
a.b.c = a.c;
```

Passing that object as a prop will then crash the method.

We are using this method in enzyme-matchers and I think it would be really confusing if anyone ever ran into this.

Reply:
Actually. Just tested and it looks like inspect does stringify circular properly! So i'll use that.

Reply:
Awesome :-) I'm not particularly worried about circular cases anyways, but glad it handles it!